doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -782,6 +782,38 @@ If you deaccession the most recently published version of the dataset but not al
 
 **Important Note**: A tombstone landing page with the basic citation metadata will always be accessible to the public if they use the persistent URL (Handle or DOI) provided in the citation for that dataset.  Users will not be able to see any of the files or additional metadata that were previously available prior to deaccession.
 
+.. _dataset-types:
+
+Dataset Types
+=============
+
+Out of the box, all datasets have a dataset type of "dataset". Superusers can add additional types such as "software" or "workflow" using the :ref:`api-add-dataset-type` API endpoint.
+
+Once more than one type appears in search results, a facet called "Dataset Type" will appear allowing you to filter down to a certain type.
+
+If your installation is configured to use DataCite as a persistent ID (PID) provider, the appropriate type ("Dataset", "Software", "Workflow") will be sent to DataCite when the dataset is published for those three types.
+
+Currently, the dataset type can only be specified via API and only when the dataset is created. For details, see the following sections of the API guide:
+
+- :ref:`api-create-dataset-with-type` (Native API)
+- :ref:`api-semantic-create-dataset-with-type` (Semantic API)
+- :ref:`import-dataset-with-type`
+
+Dataset types can be listed, added, or deleted via API. See :ref:`api-dataset-types` in the API Guide for more.
+
+Development of the dataset types feature is ongoing. Please see https://github.com/IQSS/dataverse/issues/10489 for details.
+
+.. _supported-dataset-types:
+
+Supported Dataset Types
+-----------------------
+
+Out of the box, Dataverse is configured with three dataset types:
```

Comment:
No longer true

Reply:
What if we added an "enabled" flagto the table? That way we could have all 3 in this PR and just leave dataset enabled.

Reply:
Yes, there's definitely a parallel here with licenses.

I simply deleted the "Supported Dataset Types" section from the docs in 647121acbd. Higher up in the doc we already say `Out of the box, all datasets have a dataset type of "dataset". Superusers can add additional types such as "software" or "workflow"` so the easiest path forward is to simply remove the information that's no longer accurate (I should have removed that section in 200a45a).

As for an `enabled` flag, sure maybe we can add it in a future PR.